build.sh

```diff
@@ -255,23 +255,25 @@ build_gmake() {
     cp make gmake
 }
 
-JDK_DIR=openjdk-8u72b15-bsd-port-20160220
+# We use the JDK we just built for all consequent Java compilation (and as a
+# basis for the Graal compiler).
 case `uname` in
-    Linux)
-      JDK_JAVAC=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/jdk/bin/javac;;
-    OpenBSD)
-      JDK_JAVAC=${wrkdir}/openjdk/build/bsd-x86_64-normal-server-release/jdk/bin/javac;;
+    Linux)   OUR_JAVA_HOME=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image/;;
```

Comment:
Ah wait. I see some confusion. `JAVA_HOME` is a standard Java variable used to specify which JDK to use for various Java gubbins. The particular variable `OUR_JAVA_HOME` is a variable which I am going to pass to a part of the build system which expects me to set a `JAVA_HOME`. Hence the naming I used. E.g.:

```
MX="env JAVA_HOME=${OUR_JAVA_HOME} python2.7 ${wrkdir}/mx/mx.py"
```